packages/core/ios/RNEmbrace/EmbraceManager.m

```diff
@@ -1,181 +1,137 @@
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(setUserAsPayer : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(clearUserAsPayer : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(setReactNativeVersion : (NSString *)version resolver : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(removeSessionProperty : (NSString *)key resolver : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(setUserEmail : (NSString *)userEmail resolver : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(addSessionProperty : (NSString *)key value : (NSString *)value permanent : (BOOL)
+                      permanent resolver : (RCTPromiseResolveBlock)
+                          resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(logMessageWithSeverityAndProperties : (NSString *)message severity : (NSString *)
+                      severity properties : (NSDictionary)properties stacktrace : (NSString *)
+                          stacktrace resolver : (RCTPromiseResolveBlock)
+                              resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(startView : (NSString *)viewName resolver : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(endView : (NSString *)spanId resolver : (RCTPromiseResolveBlock)
+                      resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(startSpan : (NSString *)name parentSpanId : (NSString *)
+                      parentSpanId startTimeMs : (nonnull NSNumber *)startTimeMs resolver : (
+                          RCTPromiseResolveBlock)resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(stopSpan : (NSString *)spanId errorCodeString : (NSString *)
+                      errorCodeString endTimeMs : (nonnull NSNumber *)endTimeMs resolver : (
+                          RCTPromiseResolveBlock)resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(addSpanEventToSpan : (NSString *)spanId name : (NSString *)name time : (double)
+                      time attributes : (NSDictionary)attributes resolver : (RCTPromiseResolveBlock)
+                          resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(addSpanAttributeToSpan : (NSString *)spanId key : (NSString *)
+                      key value : (NSString *)value resolver : (RCTPromiseResolveBlock)
+                          resolve rejecter : (RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(
+    recordCompletedSpan : (NSString *)name startTimeMs : (double)startTimeMs endTimeMs : (double)
```

Comment:
minor but I kind of find the previous version a bit more readable here, are there any quick configs around this we could tweak? In particular I feel we don't need the extra spaces in ` recordCompletedSpan : (NSString *)name` as well as the extra newlines/tabs

Reply:
just reverted the changes for this file and disabled formatter for .m by now, I can revisit it later on